test/fixture/return-rejected-promise.js

```diff
@@ -0,0 +1,11 @@
+import test from '../../';
+
+var Promise = require('bluebird');
+
+test(() => {
```

Comment:
Oops. This fixture isn't used anywhere. I just used it to create some of the above screenshots. It should be deleted.